bonkers.css

```diff
@@ -53,6 +53,7 @@
   --shadow-border-secondary: 0 0 0 4px rgb(223 225 233 / 50%);
   --shadow-border-primary-600: 0 0 0 2px var(--color-primary-600);
   --shadow-border-primary-700: 0 0 0 2px var(--color-primary-700);
+  --shadow-toggle-primary: 0 0 0 4px rgb(52 118 56 / 50%);
```

Comment:
im not super happy with naming... - is --shadow-border-primary is super different? 
it's better to give reusable names... which don't imply where the style will be used

<img width="869" alt="image" src="https://github.com/user-attachments/assets/63b97d23-aca2-444c-87b7-5198253f9f41" />